spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/RDDFunctions.scala

```diff
@@ -152,7 +152,8 @@ class RDDFunctions[T](rdd: RDD[T]) extends WritableToCassandra[T] with Serializa
   def joinWithCassandraTable[R](
     keyspaceName: String, tableName: String,
     selectedColumns: ColumnSelector = AllColumns,
-    joinColumns: ColumnSelector = PartitionKeyColumns)(
+    joinColumns: ColumnSelector = PartitionKeyColumns,
+    readConf: ReadConf = ReadConf.fromSparkConf(rdd.sparkContext.getConf))(
```

Comment:
should these be implicits like SparkContextFunctions#cassandraTable?

Reply:
I tried to keep this consistent with other functions in this file, which pass a defaulted WriteConf (RDDFunctions#saveToCassandra)